testing-files/perfectThing.js

```diff
@@ -168,6 +172,11 @@ srv.start().then((WoT) => {
                 })
             })
 
+            setInterval(myEventCallback, 400)
+            function myEventCallback() {
+                thing.emitEvent("onChange", 42)
+            }
+
             thing.expose().then(() => {
                 console.info(thing.title + " ready")
             })
```

Comment:
@egekorkan In my opinion perfectThing is ready. It has two observableProperties and two events one of each times out the other emits valid data. Is this sufficient?

Reply:
Yes, great work!